backend/tracim_backend/lib/core/workspace.py

```diff
@@ -178,20 +184,35 @@ def get_one_for_current_user(self, id):
     def get_all(self):
         return self._base_query().all()
 
-    def get_all_for_user(self, user: User, ignored_ids=None):
-        workspaces = []
+    def _get_workspace_owned_by_user(self, user_id: int) -> typing.List[Workspace]:
+        return self._base_query_without_roles().filter(Workspace.owner_id == user_id).all()
+
+    def _get_workspace_ids(self, workspaces: typing.List[Workspace]) -> typing.List[int]:
+        workspace_ids = []
+        for workspace in workspaces:
+            workspace_ids.append(workspace.workspace_id)
+        return workspace_ids
```

Comment:
I'm not sure specific method is useful for that. @lebouquetin What is your opinion about use short code like following instead a real function ?

```python
[w.id for w in workspaces]
```

Reply:
@buxx @inkhey this way of writing code may be complicated to understand... when it is combined with conditionnal options and sub-inlining operations.

The example you write is ok: the code is clear and easy to read and understand

Reply:
fixed.